### DISCUSSION TITLE
Why is A3 mega gke cluster bluepriny beta status?

### DISCUSSION THREAD

**Discussion by donggrame** [GoogleCloudPlatform/cluster-toolkit#3464] [Q&A] [answered]
Is it because significantly slower than slurm blueprint?

We are considering to spin up A3 mega GKE for LLM training and would like to know what could go wrong before even start it due to high expanse.

**Comment by ighosh98** [comment, marked as answer]
Hi @donggrame 
Thanks for reaching out. A3 mega GKE is GA and no longer in beta. We have a backlog task associated with updating the documentation for provisioning a GKE cluster with cluster-toolkit. Please consider filing a [Google Support Case](https://cloud.google.com/support-hub) so that we can resolve your issue quicker.

**Reply by donggrame** [reply]
Thanks for reply @ighosh98.
I may have confused you, but what I meant was the reason why blueprint for [GKE A3 Mega](https://github.com/GoogleCloudPlatform/cluster-toolkit/blob/main/examples/gke-a3-megagpu.yaml) has *experimental* label but [Slurm](https://github.com/GoogleCloudPlatform/cluster-toolkit/tree/main/examples/machine-learning/a3-megagpu-8g) example does not, which I interpreted it as *stable*.
![image](https://github.com/user-attachments/assets/8f7db529-db9c-4fce-b61a-bd88cb7342de)

I was wondering if the reason behind *experimental* label includes slower performance than Slurm due to whatever reasons I can not tell.

**Reply by ighosh98** [reply]
We should update it in one of the upcoming releases. Thanks for pointing this out!

**Reply by ighosh98** [reply]
There should be no performance issues if you use GKE A3 Mega blueprint. You could reach out to us if you see any issues.

**Reply by donggrame** [reply]
Many thanks! 👍

**Reply by ighosh98** [reply]
Hi @donggrame 
We have now published the document for provisioning A3 Mega clusters. Here is the link to it: https://cloud.google.com/cluster-toolkit/docs/deploy/deploy-a3-mega-gke-cluster
